view/preview/index.css

```diff
@@ -0,0 +1,65 @@
+.preview {
+  position: relative;
+}
+
+.preview__graphics {
+  height: 200vh;
```

Comment:
Lets add `width: 100%` / `max-width: 100%` or `overflow-x: hidden` to prevent horizontal scrollbar :)

![image](https://user-images.githubusercontent.com/22644149/203848718-da1c011c-222b-4924-b4a4-271a7541e567.png)

Reply:
added overflow